src/pks/surface_balance/constitutive_relations/land_cover/albedo_threecomponent_evaluator.cc

```diff
@@ -81,7 +85,7 @@ AlbedoThreeComponentEvaluator::Evaluate_(const State& S,
 
     for (auto c : lc_ids) {
       // albedo of the snow
-      albedo[2][c] = Relations::CalcAlbedoSnow(snow_dens[0][c]);
+      albedo[2][c] = is_constant_snow_albedo_ ? plist_.get<double>("albedo snow [-]") : Relations::CalcAlbedoSnow((*snow_dens)[0][c]); 
```

Comment:
No, also need to take this out.  All parsing of the ParameterList must be done in the constructor.  Also add

`albedo_snow_ = plist_.get<double>("...");`

in the constructor, then just use it here.